### ISSUE TITLE
Estudiar los estándares ERC20, ERC721, ERC1155

### ISSUE THREAD

**Issue by pymachado** [Blockitus/MoneyInBag#2]
https://ethereum.org/
https://github.com/OpenZeppelin/

**Comment by pymachado** [comment]
Bary -> crear una colección de un ERC721
Dayi -> crear una colección de un ERC1155

**Comment by BaryModesto** [comment]
I was looking at Open Zepelin, and I really did not find information about this specific topic.

**Comment by BaryModesto** [comment]
Navigating in the search for information, I came across OpenZepelin again, which gave me the idea that I was not looking in the right place.

**Comment by BaryModesto** [comment]
While browsing the information, I found OpenZepelin again, which gave me the idea that I wasn't looking in the right place. 

So I deepened my search and found this bookstore. From what I see it is very easy to call functions already implemented by this entity, and create the NFTs. 

Which brings me to the question, is the issue you want us to use these libraries or create from scratch the implementation of NFTs?

**Comment by BaryModesto** [comment]
[This is the link where are all documentation](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/token/ERC721)

**Comment by pymachado** [comment]
https://eips.ethereum.org/EIPS/eip-721 This is the EIP for ERC721, the point is not reinventing the wheel is learn to implemented. Because in real project with timelines we don't have time to make a new wheel. So I would like that you understand how does ER721 works? and make a collection supporting one of the implementations mentioned in the link above .

**Comment by BaryModesto** [comment]
Understand perfeclty , bro.

 I have a question: what happens if Open Zepeling delete all its library from Github? The contract of all world in EVM will not usable?

**Comment by pymachado** [comment]
No, not at all. OpenZeppelin is just a library with a bunch of implementations of smart contracts. In blockchain, contracts are not in source code, instead they are in bytecode source. And another point is OpenZeppelin just brings solutions for EIPs, and those documents are the official ethereum improvement proposals, specifying the standard interfaces core for the Ethereum's core or Subcores (smart contracts). And those has a hard decentralized process to be accepted for the community as a standard. So it is more complex than what happen if OpenZeppelin delete its repo from Github? the answer is. NOTHING HAPPEN.

**Comment by BaryModesto** [comment]
Hi, this task was completed

**Comment by pymachado** [comment]
You need to close the issue .

**Comment by BaryModesto** [comment]
I can't complete, because i am not an administrator
